major_event_log/feeds.py

```diff
@@ -1,32 +1,155 @@
 """Creates the Atom feed for the major PREMIS events."""
 
 from django.contrib.syndication.views import Feed
-from django.core.urlresolvers import reverse
-from django.utils.feedgenerator import Atom1Feed
+from django.core.urlresolvers import reverse, reverse_lazy
+from django.utils.feedgenerator import Atom1Feed, rfc3339_date
+from django.core.paginator import Paginator
```

Comment:
Should I move the two middle imports to the top, and also place both imports from django.core next to each other?